delta/plugins/elasticsearch/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/elasticsearch/model/ResourcesSearchParams.scala

```diff
@@ -57,6 +58,22 @@ final case class ResourcesSearchParams(
 
 object ResourcesSearchParams {
 
+  sealed trait TypeOperator
+
+  object TypeOperator {
```

Comment:
Wouldn't it be possible to make a general And/Or operator? The directive can still be called typeOperator. This way we don't have two ways of doing an ES "Or"

Reply:
cf. `locate` parameter

Reply:
I'm not sure I understand. Are we doing OR already in ES?

Reply:
Fixed